soccer/gameplay/plays/restarts/basic_corner_kick.py

```diff
@@ -0,0 +1,60 @@
+import main
+import robocup
+import behavior
+import constants
+
+import standard_play
+import evaluation
+import tactics.coordinated_pass
+
+#Holds a point or a line, with the associted priority(0 is highest priority)
+class PointAndPriority:
+	def __init__(self, line_pt, priority):
+		self.line_pt = line_pt
+		self.priority = priority
+
+class BasicCorner(standard_play.StandardPlay):
+	def __init__(self):
+		super().__init__(continuous = True)
+		#the minimum number we want the probability multiplier be
+		MINIMUM_MULTIPLIER = 0.5
+		refelctor = 1
+		#Depending on corner's location the point are refelcted
+		if main.ball().pos.x < 0:
+			refelctor = -1
+
+		line1 = PointAndPriority(robocup.Line(robocup.Point(-1.9 * refelctor, 7.45), 
+			robocup.Point(-1.9 * refelctor, 8.5)), 0)
+		line2 = PointAndPriority(robocup.Line(robocup.Point(-0.56 * refelctor, 7.11), 
+			robocup.Point(0.44 * refelctor, 6.2)), 1)
+		line3 = PointAndPriority(robocup.Line(robocup.Point(2.40 * refelctor, 5.85), 
+			robocup.Point(0.99 * refelctor, 5.85)), 2)
```

Comment:
In general, are the priorities for these lines going from 0 up to the number of lines - 1?